### ISSUE TITLE
Don't output diagnostics while parsing schema

### ISSUE THREAD

**Issue by bburtin** [wework/speccy#157]
## Detailed description

Speccy commands output diagnostics about the files that are being parsed:
```
GET file1.yaml
GET file2.yaml
GET file3.yaml
```
This is not very useful in most cases, and creates noise in the output of the command.  

## Context

It's particularly bad for the **resolve** command, because it results in YAML that can't be parsed without manual editing.

## Possible implementation

Ideally Speccy would either not print the list of files being parsed, or have a verbose option that prints this information to stderr.

## Your environment

Node 10, Ubuntu 18.04.

**Comment by bburtin** [comment]
I just noticed that there's a `--quiet` option to resolve, which should do what I want.  I believe that this should be the default behavior, per usual UNIX CLI convention.  Also, I couldn't get `--quiet` to work, possibly due to issue #151 .

```
$ node_modules/.bin/speccy resolve -q v2api.yaml 

  Usage: speccy <command>

  Options:

    -V, --version                    output the version number
    -c, --config [configFile]        config file (containing JSON/YAML). See README for potential values.
    -h, --help                       output usage information

  Commands:

    lint [options] <file-or-url>     ensure specs are not just valid OpenAPI, but lint against specified rules
    resolve [options] <file-or-url>  pull in external $ref files to create one mega-file
    serve [options] <file-or-url>    view specifications in beautiful human readable documentation
```

**Comment by philsturgeon** [comment]
Fixed in #161.

Run `npm install speccy@0.8.2-2` to get this fix.
